### PR TITLE
Add Instructions on how to get Dagger Engine to work on RHEL to the Docs 

### PR DESCRIPTION
**Pull request by levlaz**
This adds a section the troubleshooting doc on how to get Dagger Engine to run on RHEL. Documents the issue that was fixed here: https://github.com/dagger/dagger/issues/6995

### CODE REVIEW

File: docs/current_docs/manuals/user/932727-troubleshooting.mdx [hunk 1]
```diff
@@ -62,3 +62,15 @@ To resolve this error, remove the `DOCKER_DEFAULT_PLATFORM` variable, restart th
 DAGGER_ENGINE_DOCKER_CONTAINER="$(docker container list --all --filter 'name=^dagger-engine-*' --format '{{.Names}}')"
 docker restart "$DAGGER_ENGINE_DOCKER_CONTAINER"
 ```
+
+## Dagger Engine Restarts Due to `CNI setup error`
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/user/932727-troubleshooting.mdx:66]:
```suggestion
## Dagger restarts with a "CNI setup error"
```

File: docs/current_docs/manuals/user/932727-troubleshooting.mdx [hunk 2]
```diff
@@ -62,3 +62,15 @@ To resolve this error, remove the `DOCKER_DEFAULT_PLATFORM` variable, restart th
...
+
+Dagger Engine requires the `iptable_nat` Linux Kernel Module in order to function properly. On some Linux Distributions (including RHEL 8.x and 9.x) this module is not loaded by default.
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/user/932727-troubleshooting.mdx:68]:
```suggestion
The Dagger Engine requires the `iptable_nat` Linux kernel module in order to function properly. On some Linux distributions (including Red Hat Enterprise Linux 8.x and 9.x), this module is not loaded by default.
```

File: docs/current_docs/manuals/user/932727-troubleshooting.mdx [hunk 3]
```diff
@@ -62,3 +62,15 @@ To resolve this error, remove the `DOCKER_DEFAULT_PLATFORM` variable, restart th
...
+
+You can load this module by running `sudo modprobe iptable_nat`.
+
+If you'd like to have this module loaded by default when your machine starts, then you can add it to the `/etc/modules-load.d/modules` file with the following command:
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/user/932727-troubleshooting.mdx:72]:
```suggestion
To have this module loaded automatically on startup, add it to the `/etc/modules-load.d/modules` file with the following command:
```

**Review by vikram-dagger — CHANGES_REQUESTED** [review]

**Comment by vikram-dagger** [comment]
Can you please also add the DCO so CI passes? Thanks.

**Comment by levlaz** [comment]
> Can you please also add the DCO so CI passes? Thanks.

Yes, sorry just did that.

**Comment by levlaz** [comment]
Thanks for the suggestions @vikram-dagger - I accepted them all, this is ready for review again.

**Review by vikram-dagger — APPROVED** [review]

**Comment by vikram-dagger** [comment]
The Netlify failure is on account of the Netlify build timing out and not on account of any error in the PR. I have re-run the build manually on Netlify and it was green, so merging.

**Pull request merged**